.github/README.md

```
@@ -797,28 +800,21 @@ Run these commands to ensure your code adheres to project standards and best pra
 
 ## ☕ Supporters
 
-Fiber is an open-source project that runs on donations to pay the bills, e.g., our domain name, GitBook, Netlify, and serverless hosting. If you want to support Fiber, you can ☕ [**buy a coffee here**](https://buymeacoff.ee/fenny).
+Fiber is an open-source project that runs on donations to pay the bills, e.g., our domain name, GitBook, Netlify, and serverless hosting. If you want to support Fiber, you can ☕ [**buy a coffee here**](https://buymeacoff.ee/fenny) or sponsor the organization via ⭐ [**GitHub Sponsors**](https://github.com/sponsors/gofiber).
+
+|                                                            | User                                             | Sponsorship |
+| ---------------------------------------------------------- | ------------------------------------------------ | -------- |
+| ![](https://avatars.githubusercontent.com/u/132028505?s=25)   | [@CodeRabbit](https://www.coderabbit.ai/)           | 🔥 Fiber Guardian    |
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)